patches/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.java.patch

```diff
@@ -1,10 +1,12 @@
 --- a/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.java
 +++ b/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.java
-@@ -92,6 +_,7 @@
+@@ -91,7 +_,8 @@
+     public void render(GuiGraphics p_283479_, int p_283661_, int p_281248_, float p_281886_) {
          int i = this.leftPos;
          int j = this.topPos;
-         super.render(p_283479_, p_283661_, p_281248_, p_281886_);
-+        net.neoforged.neoforge.common.NeoForge.EVENT_BUS.post(new net.neoforged.neoforge.client.event.ContainerScreenEvent.Render.Background(this, p_283479_, p_283661_, p_281248_));
+-        super.render(p_283479_, p_283661_, p_281248_, p_281886_);
++        // Neo: replace the super render call with renderBackground and manually render the widgets in the method below.
++        this.renderBackground(p_283479_, p_283661_, p_281248_, p_281886_);
```

Comment:
The whole point of moving the implementation of the super method here is to not patch `renderBackground()` at all. Please move the event dispatch and the widget rendering loop here after the `renderBackground()` call and adjust the comment to `// Neo: replicate the super method's implementation to insert the event between background and widgets`

Reply:
fixed!